Sources/ParseLiveQuery/Client.swift

```diff
@@ -23,7 +23,7 @@ public class Client: NSObject {
     let clientKey: String?
 
     var socket: SRWebSocket?
-    var userDisconnected = false
+    public var userDisconnected = false
```

Comment:
I use this proprty in my own project to automatically reconnect to the socket after the Internet connection is available again. Otherwise I don't know any way to know whether the client is connected or not.